core/templates/dev/head/domain/feedback_thread/FeedbackThreadSummaryObjectFactory.js

```diff
@@ -17,16 +17,16 @@
    domain objects.
  */
 
-oppia.factory('FeedbackThreadObjectFactory', [function() {
+oppia.factory('FeedbackThreadSummaryObjectFactory', [function() {
   var FeedbackThread = function(status, originalAuthorId, lastUpdated,
```

Comment:
If this is now a FeedbackThreadSummary then this var (and all references to this factory) should be renamed accordingly.